content/id/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,1202 @@
+`spec.updateStrategy` dari objek API StatefulSet. Fitur ini dapat digunakan untuk
+memperbarui _image_ Container, permintaan sumber daya dan/atau pembatasan, label,
+dan anotasi Pod dalam suatu StatefulSet. Ada dua strategi pembaruan yang berlaku,
+`RollingUpdate` dan `OnDelete`.
+
+Pembaruan dengan `RollingUpdate` adalah strategi bawaan untuk StatefulSet.
+
+### Pembaruan Bertahap (RollingUpdate)
+
+Pembaruan dengan strategi `RollingUpdate` akan memperbarui semua Pod di dalam
+StatefulSet dalam urutan indeks terbalik, dengan tetap memperhatikan 
+jaminan dari StatefulSet.
+
+Lakukan _patch_ pada StatefulSet `web` dengan menerapkan `RollingUpdate`:
+
+```shell
+kubectl patch statefulset web -p '{"spec":{"updateStrategy":{"type":"RollingUpdate"}}}'
+```
+```
+statefulset.apps/web patched
+```
+
+Pada salah satu jendela terminal, _patch_ StatefulSet `web` untuk mengubah 
+_image_ Container lagi:
+
+```shell
+kubectl patch statefulset web --type='json' -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value":"gcr.io/google_containers/nginx-slim:0.8"}]'
+```
+```
+statefulset.apps/web patched
+```
+
+Pada terminal yang lain, amati Pod pada StatefulSet:
+
+```shell
+kubectl get pod -l app=nginx -w
+```
+Keluarannya akan seperti:
+```
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          7m
+web-1     1/1       Running   0          7m
+web-2     1/1       Running   0          8m
+web-2     1/1       Terminating   0         8m
+web-2     1/1       Terminating   0         8m
+web-2     0/1       Terminating   0         8m
+web-2     0/1       Terminating   0         8m
+web-2     0/1       Terminating   0         8m
+web-2     0/1       Terminating   0         8m
+web-2     0/1       Pending   0         0s
+web-2     0/1       Pending   0         0s
+web-2     0/1       ContainerCreating   0         0s
+web-2     1/1       Running   0         19s
+web-1     1/1       Terminating   0         8m
+web-1     0/1       Terminating   0         8m
+web-1     0/1       Terminating   0         8m
+web-1     0/1       Terminating   0         8m
+web-1     0/1       Pending   0         0s
+web-1     0/1       Pending   0         0s
+web-1     0/1       ContainerCreating   0         0s
+web-1     1/1       Running   0         6s
+web-0     1/1       Terminating   0         7m
+web-0     1/1       Terminating   0         7m
+web-0     0/1       Terminating   0         7m
+web-0     0/1       Terminating   0         7m
+web-0     0/1       Terminating   0         7m
+web-0     0/1       Terminating   0         7m
+web-0     0/1       Pending   0         0s
+web-0     0/1       Pending   0         0s
+web-0     0/1       ContainerCreating   0         0s
+web-0     1/1       Running   0         10s
+```
+
+Pod dalam StatefulSet diperbarui dengan urutan indeks terbalik. Pengontrol
+StatefulSet mengakhiri setiap Pod, dan menunggunya beralih menjadi Running dan Ready
```

Comment:
nit: I found there are around 28 "Running dan Ready" on this page, italicize them all.